### PR TITLE
[sp] implement "add block to current pipeline"

### PR DESCRIPTION
**Pull request by shrey-mage**
# Summary
- users can now re-add an existing block back to a pipeline via the file editor

# Tests

cc: @johnson-mage @tommydangerous @dy46

### CODE REVIEW

File: mage_ai/frontend/components/FileEditor/index.tsx [hunk 1]
```diff
@@ -118,11 +125,24 @@ function FileEditor({
...
     saveFile,
     setFilesTouched,
-    updateFile,
   ]);
 
+  const addToPipelineEl = fileExtension === FileExtensionEnum.PY && (
+    <Spacing p={1}>
+      <Button
+        onClick={() => addNewBlock({
+          type: getBlockType(file.path.split('/')) as BlockTypeEnum,
```
Comment (johnson-mage) [inline mage_ai/frontend/components/FileEditor/index.tsx:137]: try adding BlockTypeEnum to actual method

**Comment by johnson-mage** [comment]
unbold the button text

**Review by johnson-mage — APPROVED** [review]
approved after addressing comments

**Comment by shrey-mage** [comment]
> unbold the button text

@johnson-mage Updated button styling:
image

**Pull request merged**